public/logos/1877_stockholm_tramways/D.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
these are still not compressed, there's a guide on the wiki to do it

Reply:
That's a bummer, must have somehow missed that part of the guide, will use the `generate_logos.rb` script instead now that I know it exists.